src/common/main.lua

```diff
@@ -191,10 +191,10 @@ function ant.init()
 		utils.assertHasPermission(msg.From)
 
 		local name, transactionId, ttlSeconds, priority =
-			string.lower(msg["Sub-Domain"]),
-			msg["Transaction-Id"],
-			tonumber(msg["TTL-Seconds"]),
-			tonumber(msg["Priority"])
+			string.lower(msg.Tags["Sub-Domain"]),
+			msg.Tags["Transaction-Id"],
+			tonumber(msg.Tags["TTL-Seconds"]),
+			tonumber(msg.Tags["Priority"])
```

Comment:
I think we should be using msg.Thing - check createActionHandler for how we normalize incoming strings, numbers, etc.

Keep in mind, all tags are "tabbed" in the msg table.

Reply:
@dtfiedler I tend to agree